lib/apiservers/portlayer/restapi/handlers/storage_handlers.go

```diff
@@ -507,6 +521,132 @@ func (h *StorageHandlersImpl) VolumeJoin(params storage.VolumeJoinParams) middle
 	return storage.NewVolumeJoinOK().WithPayload(actualHandle.String())
 }
 
+func (h *StorageHandlersImpl) StatPath(params storage.StatPathParams) middleware.Responder {
+	defer trace.End(trace.Begin(params.DeviceID))
+
+	vc := epl.Containers.Container(params.DeviceID)
+	if vc == nil {
+		log.Debugln("%#v", spew.Sdump(epl.Containers))
```

Comment:
spewing the entire epl containers list may be too heavy here. maybe print the ID's or simply log that the container was not found here.